### DISCUSSION TITLE
C++ Wrapper for Colour?

### DISCUSSION THREAD

**Discussion by ademmler** [colour-science/colour#923] [Q&A] [answered]
Hi there,

is there an "C++ wrapper" for "Colour" existing?

thx in advance Alex

**Comment by thomasmansencal** [comment, marked as answer]
Hi Alex,

There is no C++ wrapper, however, you could possibly use `pybind11` to embed Colour in your code: https://pybind11.readthedocs.io/en/stable/advanced/embedding.html. Note that Colour requires Imageio, Numpy and Scipy (and their hard dependencies) to run. You could still mock Scipy and imageio whilst retaining some functionality but Numpy is fully required.

Cheers,

Thomas

**Reply by ademmler** [reply]
thx for this hint - I will check this out ...
